package.json

```diff
@@ -262,5 +262,8 @@
       "PR: Breaking Changes 💥": "Breaking Changes",
       "PR: Icon 💎": "New Icons"
     }
+  },
+  "engines": {
```

Comment:
We can also specify npm and node version if we want

Reply:
coool

Reply:
Node version should be handled by `.nvmrc`.